README.md

```diff
@@ -7,25 +7,51 @@ Latest build: https://metamask.github.io/api-specs/latest/openrpc.json
 
 JSON-RPC API provided by MetaMask.
 
-MetaMask dapp developers lack a formal RPC specification. This attempts to document which methods MetaMask supports in a way that's versionable, human-readable and machine-readable. It will improve the accuracy of documentation, API, and clients.
+MetaMask dapp developers lack a formal RPC specification. This attempts to document which methods MetaMask supports in a way that's versionable, human- and machine-readable. It will improve the accuracy of documentation, API, and clients.
 
-MetaMasks effectiveness as an enterprise-grade application platform depends on its ability to provide a reliable and predictable developer experience.
+MetaMask's effectiveness as an enterprise-grade application platform depends on its ability to provide a reliable and predictable developer experience.
 
 ## Contributing
 
+> Before making any significant changes to the API spec, consider whether to first submit a [MIP](https://github.com/MetaMask/metamask-improvement-proposals/tree/main) and complete the MIP [workflow](https://github.com/MetaMask/metamask-improvement-proposals/blob/main/PROCESS-GUIDE.md).
+
 Edit the `openrpc.yaml` file.
 
-You can then run `yarn run build` to generate the output file again.
+Run `yarn build` to regenerate the output file: `dist/build/openrpc.json`.
 
-To test the result, you can paste the resulting file from `dist/build/openrpc.json` into [the OpenRPC playground](https://playground.open-rpc.org/) and view the result.
+> To view the result, paste that file's contents into [the OpenRPC playground](https://playground.open-rpc.org/).
```

Comment:
```suggestion
1. Edit the API specification in the `openrpc.yaml` file.
2. Run `yarn install` if you haven't previously set up the repository. 
3. Run `yarn run build` to re-generate the output file: `dist/build/openrpc.json`.
4. To view the result, paste that file's contents into [the OpenRPC playground](https://playground.open-rpc.org/).
```

Suggestion, make into ordered list. The testing/viewing result should be a required step. Also does `yarn run build` not work for you? We should not change documented commands without explicit reason.

Reply:
sure, done locally